### PR TITLE
wip: Correct phone number

### PR DESCRIPTION
**Pull request by AliGhahremanifar**
fixed issue  #9 and create fs file for farsi texts

### CODE REVIEW

File: packages/app/src/config/fa.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export default {
+  "contact.tel": "تلفن: ",
+  "contact.telnumber": "۰۲۱ ۵۵۴۰ ۹۴۹۵",
+};
```
Comment (yassermzh) [inline packages/app/src/config/fa.ts]: We already have this in utils/fa.ts!

File: packages/app/src/pages/contact/index.tsx [hunk 2]
```diff
@@ -56,7 +57,10 @@ const Contact = () => {
         <TouchableOpacity onPress={() => Linking.openURL("tel:+982155409495")}>
           <View style={styles.row}>
             <FormattedText style={styles.content}>
-              تلفن: ۰۲۱ ۵۵۴۰۹۴۹۵
+              {messages["contact.tel"]}
```
Comment (yassermzh) [inline packages/app/src/pages/contact/index.tsx]: `<FormattedText style={} id="contact.tel" />`

it takes an `id` prop.

File: packages/app/src/config/fa.ts (hunk 1 above)
Comment (AliGhahremanifar) [reply packages/app/src/config/fa.ts]: I'm so sorry

File: packages/app/src/pages/contact/index.tsx (hunk 2 above)
Comment (AliGhahremanifar) [reply packages/app/src/pages/contact/index.tsx]: got it, thanks 👍

**Pull request merged**